microsoft/testsuites/dpdk/dpdkutil.py

```diff
@@ -20,7 +20,7 @@
 from lisa.features import NetworkInterface
 from lisa.nic import NicInfo
 from lisa.operating_system import OperatingSystem, Ubuntu
-from lisa.tools import Dmesg, Echo, Free, Lscpu, Lsmod, Lspci, Modprobe, Mount
+from lisa.tools import Dmesg, Echo, Free, Lscpu, Lsmod, Lspci, Modprobe, Mount, kernel_config
```

Comment:
I think you should be able to just import KernelConfig and call it directly below, there's some module magic in the lisa.tools module that enables it.